src/pages/pages/Home/HomePage.tsx

```diff
@@ -1,14 +1,35 @@
-import React from 'react';
-import { BlocksList, Kpi, TransactionsList } from 'components';
-import { RootState } from 'redux/store';
-import { useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+import { BlocksList, ColumnChart, Kpi, LastRewards, LineChart, RewardsCalendar, TransactionsList } from 'components';
+import { Dispatch, RootState } from 'redux/store';
+import { useDispatch, useSelector } from 'react-redux';
+import { ChartGroupType, KpiType } from 'constant';
+import { i18n } from 'utils';
+
 import Lum from './components/Lum/Lum';
 import Lumki from './components/Lumki/Lumki';
-import { KpiType } from 'constant';
 
 const HomePage = (): JSX.Element | null => {
+    const dispatch = useDispatch<Dispatch>();
+
     const blocks = useSelector((state: RootState) => state.blocks.blocks);
     const transactions = useSelector((state: RootState) => state.transactions.transactions);
+    const assetValue = useSelector((state: RootState) => state.charts.assetValue);
```

Comment:
Maybe concat these into a single `useSelector` call with destructuring to improve readability